packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -193,6 +190,11 @@ class CupertinoPicker extends StatefulWidget {
   /// A delegate that lazily instantiates children.
   final ListWheelChildDelegate childDelegate;
 
+  /// The [magnifier] widget overlaid on top of [ListWheelScrollView],
+  /// it will be vertically centered and will be the same size as the magnified item,
+  /// default to use [CupertinoPickerDefaultMagnifier].
+  final Widget magnifier;
```

Comment:
I'd rename this `selectionOverlay`.  A 'magnifier' is what it does, not what it is. We don't know what it's going to do if the user is free to specify anything they want here.

Reply:
Thank you, I've solved this now.